components/demo.vue

```diff
@@ -0,0 +1,60 @@
+<script>
+import { mapMutations } from 'vuex'
+
+export default {
+  name: 'LandingPage',
+  data() {
+    return {
+      
+    }
+  },
+  computed: {
+    todos() {
+      return this.$store?.state?.todos?.list || []
```

Comment:
We can use [mapGetters](https://vuex.vuejs.org/guide/getters.html) here.

Reply:
Ok, bhai, Thanks.